src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -97,3 +103,42 @@ case class BreakdownResult(
     delta: Double)
 
 case class Failure(className: String, message: String)
+
+// KEEP ARGUMENTS SORTED BY NAME.
+// It simplifies lookup when checking if a parameter is here already.
+case class MLParams(
+    // *** Common to all algorithms ***
+    randomSeed: Option[Int] = Some(0),
+    numExamples: Option[Long] = None,
+    numTestExamples: Option[Long] = None,
+    numPartitions: Option[Int] = None,
+    // *** Specialized and sorted by name ***
+    ldaDocLength: Option[Int] = None,
+    ldaNumVocabulary: Option[Int] = None,
```

Comment:
Can you please make the param names match those in MLlib?  It will making writing configs easier.